### DISCUSSION TITLE
404 when viewing organizations

### DISCUSSION THREAD

**Discussion by xxAROX** [community/community#61346] [Mobile] [answered]
I get a message that this organization dosnt exists on mobile. The web version it works fine.

**Comment by xxAROX** [comment]
![Screenshot_20230720-132823_GitHub.png](https://github.com/community/community/assets/57589973/e55821cb-2625-44b3-9e6f-386096225678)

**Reply by esahmed21** [reply]
Hi @xxAROX 👋

Thank you for reaching out! Would you mind sharing more details if this happens with every organisation or a specific one? It will be great if you share the organisation name so that we try reproducing that and help you!

Thanks

**Reply by xxAROX** [reply]
So it happens to some organizations, but not all

**Reply by xxAROX** [reply]
*After noticing a new update was out I updated the GitHub app, everything works fine now. thanks. 👍

**Reply by esahmed21** [reply, marked as answer]
Awesome! Glad that things are working after updating the app!

Thanks for reaching out to us!

**Reply by Kjusass** [reply]
- [ ] #-
